packages/experimental/Badge/src/PresenceBadge/PresenceBadge.tsx

```diff
@@ -35,22 +43,21 @@ export const PresenceBadge = compose<PresenceBadgeType>({
   ...stylingSettings,
   slots: {
     root: View,
-    svgXml: SvgXml,
+    svg: Svg,
   },
   useRender: (userProps: PresenceBadgeProps, useSlots: UseSlots<PresenceBadgeType>) => {
-    const badge = useBadge(userProps) as PresenceBadgeProps;
-    const Slots = useSlots(badge, (layer) => badgeLookup(layer, badge));
+    const Slots = useSlots(userProps, (layer) => prensenceBadgeLookup(layer, userProps));
 
     return (final: PresenceBadgeProps) => {
-      const { size, status, outOfOffice, ...mergedProps } = mergeProps(badge, final);
+      const { size, status, outOfOffice, ...mergedProps } = mergeProps(userProps, final);
```

Comment:
Why is this being changed to userProps? Wouldn't useBadge modify the props so we should want to use those as the base for mergeProps instead?

Reply:
There are mostly props which are used in common badge, e.g. iconPosition. But I think you're right, I'd better return it. Because not all the functionality of Badge is implemented